webservices/schemas.py

```diff
@@ -461,14 +461,7 @@ class CandidateHistoryTotalSchema(schemas['CandidateHistorySchema'], schemas['Ca
             'contributor_employer_text',
             'contributor_occupation_text',
         ),
-        'relationships': [
-            Relationship(
-                models.ScheduleA.committee,
-                models.CommitteeHistory.name,
-                'committee_name',
-                1
-            ),
-        ],
+
```

Comment:
Ah, yeah, this will need to be added back in as well for the regular Schedule A itemized data too.  I just did a test and here's what you see without it:

<img width="1002" alt="screen shot 2017-01-13 at 2 42 29 pm" src="https://cloud.githubusercontent.com/assets/121565/21943427/36513898-d99f-11e6-9c9f-0d947b1fe14e.png">

vs. this, with the relationship:

<img width="986" alt="screen shot 2017-01-13 at 2 45 06 pm" src="https://cloud.githubusercontent.com/assets/121565/21943447/4260a8e4-d99f-11e6-8c6b-59b462160dc5.png">

It may be a bit hard to see, but the committee name column is there.